# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/OculusSession.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/vr/service/OculusSession.cpp
--- a/gfx/vr/service/OculusSession.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/vr/service/OculusSession.cpp	Fri May 24 11:32:54 2019 +0000
@@ -11,7 +11,6 @@
 #include <math.h>
 #include <d3d11.h>
 
-#include "gfxPrefs.h"
 #include "mozilla/StaticPrefs.h"
 #include "mozilla/dom/GamepadEventTypes.h"
 #include "mozilla/dom/GamepadBinding.h"
@@ -207,7 +206,7 @@
 OculusSession::~OculusSession() { Shutdown(); }
 
 bool OculusSession::Initialize(mozilla::gfx::VRSystemState& aSystemState) {
-  if (!gfxPrefs::VREnabled() || !gfxPrefs::VROculusEnabled()) {
+  if (!StaticPrefs::VREnabled() || !StaticPrefs::VROculusEnabled()) {
     return false;
   }
 